testing/e2e/suites/utils.go

```diff
@@ -28,7 +28,7 @@ func waitForAvailable(ctx context.Context, c client.Client, deployment appsv1.De
 			}
 		}
 
-		if time.Since(start) > 10*time.Minute { // 10 minutes because it takes a decent amount of time for dns to "propagate"
+		if time.Since(start) > 20*time.Minute { // 20 minutes because it takes a decent amount of time for dns to "propagate"
```

Comment:
leave the context you told me where this is no longer about dns propagating. it's about identities too

Reply:
done